text/0004-configurational-dependencies.md

```diff
@@ -49,6 +49,8 @@ These dependencies will be installed as early as possible in order to be able to
 
 ## Prior Art
 
+There was a big RFC by Brandon Cheng about a much more powerful system using [templates](https://github.com/pnpm/rfcs/pull/3), which was rejected.
```

Comment:
The new system here targeting just pnpm specific settings is a lot easier to understand. 🙂

Reply:
TBH, I don't remember why exactly it was rejected.